src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Semmle"
+      },
+      "results": [
+        {
+          "level": "warning",
+          "message": "Avoid using thread priorities. The effect is unpredictable and not portable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileCleaningTracker.java",
+                "region": {
+                  "startLine": 211,
+                  "startColumn": 13,
+                  "endLine": 211,
```

Comment:
It was an easy change, so I fixed this.

---
In reply to: [90075704](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90075704) [](ancestors = 90075704)